services/frontend/gen_assets.go

```diff
@@ -760,14 +759,14 @@ tVIBAA==
 
 	"/": {
 		name:  "/",
-		local: `.`,
+		local: `services/frontend/web_assets`,
 		isDir: true,
 	},
 }
 
 var _escDirs = map[string][]os.FileInfo{
 
-	".": {
+	"services/frontend/web_assets": {
```

Comment:
This and the change on line 762 need to be reverted. This was why I didn't keep the `esc` command as-is in the build file. Not really sure why this changes even though we specify `--prefix` but never looked into it closely. Rest lgtm.

Reply:
It seems to work. Maybe this is just internal metadata that doesn't affect serving. Did you see any effects from this before?